site/content/docs/standard/content-styles/hover-effects/index.html

```diff
@@ -50,6 +50,32 @@
   {{< /twsnippet/demo >}}
 
 
+  <!-- Section: Note about usage with animation  -->
+  <section>
+    <h3>Showing the animation effect</h3>
```

Comment:
Please add the following class to h3 element to be consistent with the rest of the document: `mb-3 text-2xl font-semibold leading-normal`.